### PR TITLE
Added README.md in ArduinoLowPower

### PR DESCRIPTION
**Pull request by Engineer1999**
I am a GSoC 202 aspirant. During the exploration of library codebase,  I found this repository which doesn't contain the README file. So, I have added a README file in ArduinoLowPower.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Low Power library
+
+This library allows user to use the low power feature of the SAMD21 MCU. This means your battery powered projects will have a loonger battery life on boards like [MKRZero](https://store.arduino.cc/usa/arduino-mkrzero), [MKR1000](https://www.arduino.cc/en/Main/ArduinoMKR1000) and [MKRFox1200](https://www.arduino.cc/en/Main/ArduinoBoardMKRFox1200).
```
Comment (per1234) [inline README.md]:
```suggestion
This library allows the use of the low power features of the SAMD21 MCU. This means your battery powered projects will have a longer battery life on boards like [MKRZero](https://store.arduino.cc/usa/arduino-mkrzero), [MKR1000](https://www.arduino.cc/en/Main/ArduinoMKR1000) and [MKRFox1200](https://www.arduino.cc/en/Main/ArduinoBoardMKRFox1200).
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Low Power library
```
Comment (per1234) [inline README.md]:
```suggestion
# Arduino Low Power library
```
The official library name is defined at https://github.com/arduino-libraries/ArduinoLowPower/blob/c1b24fb45651b5eef859ae6fb0da76c19560f2ff/library.properties#L1
This is the name to search for in the Arduino Library Manager index whether using the Arduino IDE, Arduino Pro IDE, or arduino-cli (`arduino-cli lib install "Arduino Low Power"`)

**Comment by Engineer1999** [comment]
hey @per1234 , can you please share how can I connect with you? I am a GSoC2020 aspirant and I am targeting Arduino for GSoC2020. 
Would it be possible for you to give advice about my proposal?
My linkedIn- https://www.linkedin.com/in/bhargav-p-patel/

**Comment by per1234** [comment]
@Engineer1999 please open an issue at https://github.com/arduino/summer-of-code/issues with an introduction of yourself, as recommended in our [Application Instructions](https://github.com/arduino/summer-of-code/blob/master/how-to-apply.md#during-the-application-phase-february-20th---march-31st). I see you did open an issue already, but that was more of a specific question than an introduction, and is closed anyway.

We can use your dedicated issue in that repository as a place for discussion. That is the best approach because it will allow any of Arduino administrators and mentors with the program to participate and see what has been discussed.

Although I am an administrator with the program, I'm in more of a role of just trying to help out where I can. Some of the others from Arduino could probably provide more authoritative advice on your proposal.

**Review by aentinger — APPROVED** [review]
Thank you @Engineer1999 🚀

**Pull request merged**